### DISCUSSION TITLE
Command 'pnpm dev' failed !!!!

### DISCUSSION THREAD

**Discussion by alexlili** [scalar/scalar#3079] [Q&A]
when I finish installation and input 'pnpm dev', there is an error: build-tooling/node_modules/.bin/rollup: 16: exec: node: not found

**Comment by marclave** [comment]
hi @alexlili , do you have node installed? :) 

if you type 

```
node -v
```
what is the output? :)

**Reply by alexlili** [reply]
20.16.0

**Reply by alexlili** [reply]
![Screenshot 2024-09-11 164137](https://github.com/user-attachments/assets/62e5f6d8-3c32-4388-b791-8dc50c7cf7eb)

**Reply by alexlili** [reply]
God, I hate windows system. I do not know how to handle this problem because it works in my PC - Macbook

**Reply by alexlili** [reply]
> hi @alexlili , do you have node installed? :)
> 
> if you type
> 
> ```
> node -v
> ```
> 
> what is the output? :)

How to handle this problem? Help...... I have uninstalled node, pnpm and nvm and installed them again and again. But it does not work.

**Comment by alexlili** [comment]
Hi, Marc

I have resolved this problem. Can I ask you another question? My boss asked me to use scalar in .NET project, but I found the .NET project is using old core version - 4.8. I  do not know how to integrate scalar into it. Could you send me a documentation. In our project, there is no startup.cs file and no program.cs, I need to change code in global.asax.cs file. I do not know how to handle it.

Please.

I am begging you.

Best wishes

Alex
________________________________
